packages/gestalt/src/ActivationCard.js

```diff
@@ -0,0 +1,147 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from './Box.js';
+import Heading from './Heading.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Button from './Button.js';
+import Text from './Text.js';
+import { useColorScheme } from './contexts/ColorScheme.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+import styles from './ActivationCard.css';
+
+type LinkData = {|
+  accessibilityLabel?: string,
+  href: string,
+  label: string,
+  onClick?: AbstractEventHandler<
+    | SyntheticMouseEvent<HTMLButtonElement>
+    | SyntheticMouseEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLButtonElement>
+  >,
+|};
+
+type Props = {|
+  dismissButton?: {|
+    accessibilityLabel: string,
+    onDismiss: () => void,
+  |},
+  message: string,
+  linkData?: LinkData,
+  status: 'notStarted' | 'pending' | 'needsAttention' | 'complete',
+  statusMessage: string,
+  title?: string,
+|};
+
+const STATUS_ICONS = {
+  notStarted: undefined,
+  pending: { symbol: 'clock', color: 'gray' },
+  needsAttention: { symbol: 'workflow-status-problem', color: 'red' },
+  complete: { symbol: 'check-circle', color: 'green' },
```

Comment:
Would love some feedback/recommendation on how to handle the "complete" status. The design feels significantly different from the other statuses to me. I wonder if it would be better to have a separate return statement or if I should use a lot of have a lot of `isComplete ?` ternaries through current return statement.

<img width="598" alt="Screen Shot 2020-09-29 at 11 38 27 AM" src="https://user-images.githubusercontent.com/46180742/94601424-56acd400-0248-11eb-9afe-ff9cf8f1d243.png">

Reply:
what about rendering something like:
` {isCompleted  ? <Uncompleted href={href} etc. > : <Completed >}`